src/Uno.UI/UI/Xaml/Controls/DropDownButton/DropDownButton.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class DropDownButton : Button
+	{
+		public DropDownButton()
+		{
+			DefaultStyleKey = typeof(PersonPicture);
+		}
+
+		protected override AutomationPeer OnCreateAutomationPeer()
+		{
+			return new DropDownButtonAutomationPeer(this);
+		}
+
+		//  - Missing
```

Comment:
Why is it missing ?

Reply:
The code uses revokers - translating this to C# I would do subscribe and unsubscribe events - but the code will differ a bit (I will need to subscribe to `PropertyChanged` and use `OldValue`/`NewValue`. Is that ok or have you solved a similar situation differently?